rootfs/app/migrations/000048_add_runbooks_table.up.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN;
+
+SET search_path TO private;
+
+CREATE TABLE runbooks (
+  id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+  org_id UUID NOT NULL REFERENCES orgs (id),
+  repository_configs JSONB,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NOT NULL DEFAULT NOW(),
+
+  UNIQUE (org_id)
+);
+
+CREATE TABLE runbook_rules (
+  id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+  org_id UUID NOT NULL REFERENCES orgs (id),
+  name TEXT NOT NULL,
+  description TEXT,
+  user_groups TEXT[] NOT NULL,
+  connections TEXT[] NOT NULL,
```

Comment:
This could result in a very long list if the user has many connections. Have you considered this before creating this type of association?

For other features (guardrails, datamasking), we've been using associative tables to manage relationships. Don't you think that would be a better approach here as well? With the current structure, it will be cumbersome to perform modifications in this structure.